gsoc/settings.py

```diff
@@ -242,3 +242,7 @@
 BLOG_MULTISITE = True
 
 INTERNAL_IPS = ('127.0.0.1',)
+
+BOT_NICK = "CommandBot"
+IRC_SERVER = "irc.freenode.org"
+IRC_CHANNEL = "#channel-1"
```

Comment:
You don't need to worry about joining a channel, this will pm the bot directly and then disconnect.